app/lib/frontend/templates/views/pkg/package_list.dart

```diff
@@ -45,8 +45,10 @@ d.Node _sdkLibraryItem(SdkLibraryHit hit) {
   ].join(' • ');
 
   return _item(
+    screenshots: [],
     url: hit.url!,
     name: hit.library!,
+    version: hit.version,
```

Comment:
This is not related to screenshots - right?

@isoos is there a reason we did not have the version check here before?

Reply:
It seems to be that this is required for the screenshot URL. Unfortunately, I don't think that `version` will be provided for Flutter SDK, but in such cases it won't matter as we will not have screenshots for such hits...